configs/ssg_idm-unwanted.yaml

```
@@ -5,6 +5,8 @@ data:
   description: Packages that the Identity Management subsystem does not want to distribute in RHEL 9
   maintainer: ssg_idm
   unwanted_packages:
+    # Removed in RHEL 10
+    - python-netifaces
```

`unwanted_packages` are RPMs, where `unwanted_source_packages` are SRPMs.  While the latter is preferred, in the meantime this needs to be:
```suggestion
    - python3-netifaces
```

---

Fixed. Thank you for the info.